client/app/components/empty-state/EmptyState.jsx

```diff
@@ -200,9 +200,9 @@ function EmptyState({
         </div>
       </div>
       {closable && (
-        <a className="close-button" onClick={onClose}>
+        <Link className="close-button" role="button" aria-label="Close" onClick={onClose}>
```

Comment:
If possible this should be a `<Button type="link" icon={<CloseOutlinedIcon />} />`